src/Engine/Audio/SteamAudio.cpp

```diff
@@ -41,7 +60,28 @@ void SteamAudio::SetPlayer(IPLVector3 pos, IPLVector3 dir, IPLVector3 up) {
 }
 
 void SteamAudio::CreateRenderers(IPLhandle environment) {
-    if (renderers)
-        delete renderers;
-    renderers = new SteamAudioRenderers{ environment };
+    Clear();
+
+    this->environment = environment;
+
+    IPLRenderingSettings settings;
+    settings.frameSize = CHUNK_SIZE;
+    settings.samplingRate = SAMPLE_RATE;
+    settings.convolutionType = IPL_CONVOLUTIONTYPE_PHONON;
+
+    IPLerror err = iplCreateEnvironmentalRenderer(IPLContext{ nullptr,nullptr,nullptr }, environment, settings, outputFormat, NULL, NULL, &envRenderer);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)